js/index.js

```diff
@@ -0,0 +1,152 @@
+const siteContent = {
+  "nav": {
+    "nav-item-1": "Services",
+    "nav-item-2": "Product",
+    "nav-item-3": "Vision",
+    "nav-item-4": "Features",
+    "nav-item-5": "About",
+    "nav-item-6": "Contact",
+    "img-src": "img/logo.png"
+  },
+  "cta": {
+    "h1": "DOM Is Awesome",
+    "button": "Get Started",
+    "img-src": "img/header-img.png"
+  },
+  "main-content": {
+    "features-h4":"Features",
+    "features-content": "Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "about-h4":"About",
+    "about-content": "About content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "middle-img-src": "img/mid-page-accent.jpg",
+    "services-h4":"Services",
+    "services-content": "Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "product-h4":"Product",
+    "product-content": "Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "vision-h4":"Vision",
+    "vision-content": "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+  },
+  "contact": {
+    "contact-h4" : "Contact",
+    "address" : "123 Way 456 Street Somewhere, USA",
+    "phone" : "1 (888) 888-8888",
+    "email" : "sales@greatidea.io",
+  },
+  "footer": {
+    "copyright" : "Copyright Great Idea! 2018"
+  },
+};
+
+
+// ~~~~~ Header ~~~~~
+const a1 = document.createElement("a");
+document.querySelector('nav').appendChild(a1); // Appending
+const a2 = document.createElement("a");
+document.querySelector('nav').prepend(a2); // Prepending
+
+let headerA = document.querySelectorAll('nav a');
+headerA[0].innerHTML = "Prepend";
+headerA[1].innerHTML = siteContent['nav']['nav-item-1'];
+headerA[2].innerHTML = siteContent['nav']['nav-item-2'];
+headerA[3].innerHTML = siteContent['nav']['nav-item-3'];
+headerA[4].innerHTML = siteContent['nav']['nav-item-4'];
+headerA[5].innerHTML = siteContent['nav']['nav-item-5'];
+headerA[6].innerHTML = siteContent['nav']['nav-item-6'];
+headerA[7].innerHTML = "Append";
+
+headerA[0].addEventListener('click',()=> {
```

Comment:
Great work with these -- looks like you got experience with lots of different types of event listeners while getting the reps in with similar format/structure.